### ISSUE TITLE
SQL of Query is wrong, when "FormatSQL" is checked

### ISSUE THREAD

**Issue by Tanarri** [joyfullservice/msaccess-vcs-addin#485]
Hi all,

I am using latest dev version 4.0.32 and have following issue on exporting a special, simple query.
The setting "Options --> Export --> Format SQL" has to be set, to reproduce this error.

- Create a test-query with the SQL:
```
SELECT "" AS Plzpostf, "" AS Ort, MSysObjects.Connect
FROM MSysObjects
WHERE (((MSysObjects.Connect)="something"));

```
- Export the sourcecode
- The SQL will be formated and saved in the .sql file
```
SELECT
  "" AS Plzpostf,
  "" AS
  O rt,
  MSysObjects.Connect
FROM
  MSysObjects
WHERE
  (
    (
      (MSysObjects.Connect)= "something"
    )
  );
```

Please note that the name of the second column is exported incorrectly as "O rt" (Space after the "O")
I have had this problem for several VCS versions, and am not sure if the error occurred in all old VCS versions.

Greetings
TanarRi

**Comment by joyfullservice** [comment]
Thanks for reporting this! I have reproduced it on my side, and am looking into why this alias is not being tokenized correctly in the formatter.

**Comment by joyfullservice** [comment]
@Tanarri - This should be fixed as of 34afcae. There was a problem with the `GetNextWords()` function where some incorrect index positions were causing an incomplete string to be returned. In this particular case the alias `Ort` was missing the last character, which turned it into `Or`, which is a reserved newline word. That's why it broke the tokenizing of this SQL string.

Anyway, it is all fixed now. Let me know if you encounter any further issues!

**Comment by Tanarri** [comment]
@joyfullservice 
Great! The issue is fixed!
I had about 8 SQL Queries which are now corrected. The token `or` and the token `Int` where affected in my queries.

Thank you for the fast correction
